packages/bulk-edit/src/browser/style/bulk-edit.css

```diff
@@ -0,0 +1,65 @@
+/********************************************************************************
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+ .theia-bulk-edit-container {
+    font-size: var(--theia-ui-font-size1);
+}
+
+.theia-bulk-edit-container .bulkEditNode,
+.theia-bulk-edit-container .bulkEditInfoNode {
+    display: flex;
+    align-items: center;
+}
+
+.theia-bulk-edit-container .bulkEditNode,
+.theia-bulk-edit-container .bulkEditInfoNode {
+    width: calc(100% - 32px);
+}
+
+.theia-bulk-edit-container .bulkEditNode div,
+.theia-bulk-edit-container .bulkEditInfoNode div {
+    margin-right: 5px;
+}
+
+.theia-bulk-edit-container .bulkEditInfoNode .name,
+.theia-bulk-edit-container .bulkEditInfoNode .path {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.theia-bulk-edit-container .bulkEditInfoNode .path {
+    font-size: var(--theia-ui-font-size0);
+    color: var(--theia-descriptionForeground);
+    align-self: flex-end;
+    white-space: nowrap;
+}
+
+.theia-bulk-edit-container .bulkEditNode .message {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.theia-bulk-edit-container .bulkEditNode .message .replaced-text {
+    text-decoration: line-through;
+    background-color: #4B1818;
```

Comment:
We should instead define variables for color, like 
```css
color: var(--theia-descriptionForeground);
```

So that it will work nicely for all themes. If there is no color available then we can include a new `ColorContribution` for this extension.

Reply:
@vince-fugnitto 
Tried to find where **--theia-descriptionForeground** is defined and can't seem to find it (I copied styling from callhierarchy extension). Could you direct me to it?

Reply:
It should come directly from `monaco`, if the color does not exist for your use-case (similar to vscode), then a color contribution is required: https://github.com/microsoft/monaco-editor/blob/master/test/playground.generated/customizing-the-appearence-exposed-colors.html#L66

You can reference the color documentation when theming support was added: https://github.com/eclipse-theia/theia/pull/6475/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR9

Reply:
Fixed. Thanks